modules/core/test/v2/integration/coins/eth.ts

```diff
@@ -6,7 +6,7 @@ const co = Promise.coroutine;
 import request = require('supertest-as-promised');
 const BN = require('ethereumjs-util').BN;
 
-const expressApp = require('../../../../src/expressApp');
+const expressApp = require('../../../../../express/src/expressApp');
```

Comment:
why does this need an express app? This should probably move to express if this is testing express.